games-emulation/pcsx2/pcsx2-9999.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-PLOCALES="ar_SA ca_ES cs_CZ de_DE es_ES fi_FI fr_FR hr_HR hu_HU id_ID it_IT ja_JP ko_KR ms_MY nb_NO pl_PL pt_BR ru_RU sv_SE th_TH tr_TR zh_CN zh_TW"
+EAPI=6
+PLOCALES="ar_SA ca_ES cs_CZ da_DK de_DE es_ES fi_FI fr_FR hr_HR hu_HU id_ID it_IT ja_JP ko_KR ms_MY nb_NO pl_PL pt_BR ru_RU sv_SE th_TH tr_TR zh_CN zh_TW"
```

Comment:
Kill l10n.eclass, please.

Reply:
Done. The way I used locales by using cleaning them from predefined list is very weird.
If your reasons for removing it is different I will be grateful if your explaint them.

Reply:
We don't clean them anymore. This is matter either for upstream LINGUAS support, or INSTALL_MASK.